Jenkinsfile

```diff
@@ -53,8 +53,8 @@ SUPPORTED_STAGES = [
 ]
 
 // Supported VM Images
-DEBIAN_JESSIE = 'debian-8'
-SLAVE_IMAGE = 'gcr.io/endpoints-jenkins/debian-8:0.7'
+DEBIAN_JESSIE = 'debian-9'
```

Comment:
Running docker pull on debian-9 failed but running docker pull on debian-8 succeeds: 
~$ docker pull gcr.io/endpoints-jenkins/debian-9
Using default tag: latest
Error response from daemon: manifest for gcr.io/endpoints-jenkins/debian-9:latest not found
leitang[06:56:48 Mon Aug 13]:~/develop
~$ docker pull gcr.io/endpoints-jenkins/debian-8
Using default tag: latest
latest: Pulling from endpoints-jenkins/debian-8
Digest: sha256:b4553235129076d903ab8b5047c7170d3b3b22c7cf93ec0282bf96cd99045d9c
Status: Image is up to date for gcr.io/endpoints-jenkins/debian-8:latest

Reply:
you can try
only change DEBIAN_JESSIE = 'debian-9'
but don't change SLAVE_IMAGE